metricbeat/module/system/raid/_meta/docs.asciidoc

```diff
@@ -1,3 +1,5 @@
 === system raid MetricSet
```

Comment:
This line seems to be the cause of the failing doc generation task in CI. I think it should be removed.

Reply:
agree. will remove it.